resteasy-jaxrs/src/main/java/org/jboss/resteasy/core/interception/jaxrs/AbstractReaderInterceptorContext.java

```diff
@@ -55,7 +55,7 @@ public Object proceed() throws IOException
              LogMessages.LOGGER.debugf("MessageBodyReader: %s", reader.getClass().getName());
          return readFrom(reader);
       }
-      LogMessages.LOGGER.debugf("Interceptor: %s", interceptors[index].getClass().getName());
+      LogMessages.LOGGER.debugf("ReaderInterceptor: %s", interceptors[index].getClass().getName());
```

Comment:
Changing the print message of https://github.com/resteasy/Resteasy/pull/1008/files#diff-66ed509af1b678e5645b712c6dfee91fR58. Why not.